commands/store.go

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) Alex Ellis 2017. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"text/tabwriter"
+
+	"github.com/openfaas/faas-cli/proxy"
+	"github.com/spf13/cobra"
+)
+
+var storeAddress string
+
+const defaultStore = "https://cdn.rawgit.com/openfaas/store/master/store.json"
+
+type storeItem struct {
```

Comment:
I wonder if we should move this into the `schema` or a similar package? @johnmccabe @rgee0